### PR TITLE
first_view edited

### PR DESCRIPTION
**Pull request by ekramteyeb**
First View edited

### CODE REVIEW

File: views/first_view.pug [hunk 1]
```diff
@@ -6,9 +6,9 @@ html
     h1 First View
     p Hello, World!
     p This is the first view.
-    p
-      a(href='/second_view') Second View
-    p
-      a(href='/') Home
-    p
-      a(href='/logout') Logout
```
Comment (mesfint) [inline views/first_view.pug:14]: good job

File: views/first_view.pug [hunk 2]
```diff
@@ -6,9 +6,9 @@ html
     h1 First View
     p Hello, World!
     p This is the first view.
-    p
-      a(href='/second_view') Second View
-    p
-      a(href='/') Home
```
Comment (mesfint) [inline views/first_view.pug:12]: change this line

Comment (ekramteyeb) [reply views/first_view.pug:12]: got it

File: views/first_view.pug (hunk 1 above)
Comment (ekramteyeb) [reply views/first_view.pug:14]: 👍

**Pull request merged**